egs2/ami/vad1/scripts

```diff
@@ -0,0 +1 @@
+../asr1/scripts
```

Comment:
This is OK, but why not link to the template one (as the setup.sh should automatically link them?)